core/helpers/EEH_Form_Fields.helper.php

```diff
@@ -343,11 +343,9 @@ public static function get_form_fields_array($fields)
      */
     private static function adminHidden($class, $id, $name, $value)
     {
-        $id    = esc_attr($id);
-        $name  = esc_attr($name);
-        $class = esc_attr($class);
         return "
-        <input name='{$name}' type='hidden' id='{$id}' class='{$class}' value='{$value}' />";
+        <input name='" . esc_attr($name) . "' type='hidden' id='" . esc_attr($id) . "' class='" . esc_attr($class) . "' 
+        value='" . esc_attr($value) . "' />";
```

Comment:
these are getting returned, not echoed, so this is fine as long as it doesn't interfere with any escaping that happens downstream...